features/steps/user_initialisation_steps.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=E0102, E0602
+from behave import *
+from selenium.webdriver.support.ui import Select
+
+from features.fixtures import get_chrome_browser
+from features.steps.helpers import sign_in
+from users.factories import AdminUserFactory
+from users.factories import UserFactory
+
+
+@given("an admin exists in database")
+def step_impl(context):
+    context.admin = AdminUserFactory()
+
+
+@when("the user signs up")
+def step_impl(context):
+    # TODO: HAVE TO SOLVE CAPTCHA AND EMAIL VERIFICATION ISSUES
+    context.user = UserFactory()
+    sign_in(context)
+
+
+@when("admin changes user's employee type to '{user_type}'")
+def step_impl(context, user_type):
+    other_browser = get_chrome_browser()
+    sign_in(context=context, login=context.admin.email, browser=other_browser)
+    other_browser.find_element_by_link_text("Employees").click()
+    table = other_browser.find_element_by_class_name("table")
+    button = table.find_element_by_xpath(".//tbody/child::tr[3]/child::td[7]//a")
+    button.click()
+    user_type_selection = Select(other_browser.find_element_by_name("user_type"))
+    user_type_selection.select_by_visible_text(user_type)
+    update = other_browser.find_element_by_id("opener_user_account_update")
+    update.click()
+    confirm = other_browser.find_element_by_id("id_confirm_update_button")
+    confirm.click()
+
+    other_browser.quit()
+
+
+@then("the user should have an employee status")
+def step_impl(context):
```

Comment:
the three steps implementations look very similar to each other (especially `manager` and `admin`). Maybe we could have them extracted to a generalized function?

Reply:
The problem is I'm also checking the access to key exclusive functionalities (`Projects` link for `Manager` and `Employees` for `Admin`), so it's hard to pass those arguments in step name. Unless I extract this part to separate step? Like:
```
the user should have "Manager" status
And the user should have access to "Projects" page
```
I guess it's the quickest and cleanest solution. That way we can also easily test admin's access to `Projects` if we wanted to.

Reply:
Done, applied the concept above